metaflow/extension_support.py

```diff
@@ -141,22 +173,22 @@ def multiload_globals(modules, dst_globals):
 
 def multiload_all(modules, extension_point, dst_globals):
     for m in modules:
-        # Note that we load aliases separately (as opposed to ine one fell swoop) so
+        # Note that we load aliases separately (as opposed to in one fell swoop) so
         # modules loaded later in `modules` can depend on them
         lazy_load_aliases(
             alias_submodules(m.module, m.tl_package, extension_point, extra_indent=True)
         )
         load_globals(m.module, dst_globals)
 
 
-_py_ver = sys.version_info[0] * 10 + sys.version_info[1]
+_py_ver = sys.version_info[0] * 100 + sys.version_info[1]
 _mfext_supported = False
 
-if _py_ver >= 34:
+if _py_ver >= 304:
```

Comment:
```suggestion
if sys.version_info[:2] >= (3, 4):
```

Reply:
I'll make that change (and others related to it).